lib/appsignal/hooks/puma.rb

```diff
@@ -33,5 +37,59 @@ def stop_workers
         end
       end
     end
+
+    class PumaProbe
+      attr_reader :hostname
```

Comment:
can we move this below `private` so it's a private reader. No need for it to be public